src/actions/channel.js

```diff
@@ -106,17 +107,17 @@ export const loadChannelMembers = (isInitialLoading, after) => (
       limit,
       after,
     })
-    .then(({ results, total }) => ({
+    .then(({ results }) => ({
       users: results.map(normalizeUserData),
-      total,
     }))
-    .then(({ users, total }) => {
-      const { users: loadedUsers } = channelMembersSelector(getState())
+    .then(({ users }) => {
+      if (users.length !== limit || users.length === 0) {
```

Comment:
@AleshaOleg can you make it `users.length < limit`. It doesn't change much, but makes it clearer to read.